ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -239,44 +250,49 @@ export default class ExperimentalTab extends PureComponent {
 
     return (
       <>
-        <Typography
-          variant={TypographyVariant.H4}
+        <Text
+          variant={TextVariant.headingSm}
+          as="h4"
           color={TextColor.textAlternative}
           marginBottom={2}
-          fontWeight={FONT_WEIGHT.BOLD}
+          fontWeight={FontWeight.Bold}
         >
           {t('privacy')}
-        </Typography>
-        <div
+        </Text>
+        <Box
           ref={this.settingsRefs[1]}
           className="settings-page__content-row settings-page__content-row-experimental"
+          marginBottom={3}
         >
           <div className="settings-page__content-item">
             <span>{t('transactionSecurityCheck')}</span>
             <div className="settings-page__content-description">
-              <Typography
-                variant={TypographyVariant.H6}
+              <Text
+                variant={TextVariant.bodySm}
+                as="h6"
```

Comment:
In an attempt to lower the bar for entry for our external contributors, I may have unintentionally made it look like the `as` prop was a required prop or "best practice" for the `<Text>` component when it is completely optional. There are many instances of `<Text as="h6"` in the code base which was a result of external contributors swapping the deprecated `<Typography>` with `<Text>` allowing external contributors to make these updates without having to update any e2e tests. I now realize that was a mistake. There probably aren't many times when an `<h6>` element is appropriate. It is totally fine to use the `Text` component without the `as` prop unless it is intentional. Most of the time the default rendered HTML element should be sufficient.

Reply:
Thanks for the information. I always thought `as` is mandatory ~ will remove now 🙏🏻